### PR TITLE
course creation - first pass

### PR DESCRIPTION
**Pull request by jazztpt**
This is just a working version where I used the same type of csrf_exempt view that I have in the lms view (styled after TJ's Infusionsoft code).  I intend to convert this into a full api using Django Rest Framework -- which is already installed and used for all the internal edX apis.

### CODE REVIEW

File: cms/djangoapps/appsembler/views.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            store_for_new_course = modulestore().default_modulestore.get_modulestore_type()
+            org = "AppsemblerX"
+            number = "{}101".format(user.username)
+            run = "CurrentTerm"
+            fields = {"display_name":"Your First Course"}
+            new_course = create_new_course_in_store(store_for_new_course, user, org, 
+                                                    number, run, fields)
+
+        except:
+            import pdb; pdb.set_trace()
```
Comment (natea) [inline cms/djangoapps/appsembler/views.py]: should probably remove this pdb trace

**Comment by jazztpt** [comment]
right -- still in dev mode, but I should remove that stuff before submitting a PR

File: cms/djangoapps/appsembler/views.py (hunk 1 above)
Comment (jazztpt) [inline cms/djangoapps/appsembler/views.py]: right -- still in dev mode, but I should remove that stuff before submitting a PR

File: cms/djangoapps/appsembler/views.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            message = "Unable to create new course."
+            logger.error(message)
+
+        # course_key = unicode(new_course.id,
+        new_course_url = reverse_course_url('course_handler', new_course.id)
+        userdata = { 'userid': user.id, 'new_course_url': new_course_url }
+        data = simplejson.dumps(userdata)
+        return HttpResponse(data, status=200, content_type='application/json')
+
+    except Exception as exc:
```
Comment (bezidejni) [inline cms/djangoapps/appsembler/views.py]: Doesn't really matter that much, but usually `e` is used as the exception variable in python.

File: cms/djangoapps/appsembler/views.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+from xmodule.modulestore.django import modulestore
+
+
+logger = logging.getLogger(__name__)
+APPSEMBLER_EMAIL = 'support@appsembler.com'
+
+
+@csrf_exempt
+def create_course_endpoint(request):
+    if request.method != 'POST':
```
Comment (bezidejni) [inline cms/djangoapps/appsembler/views.py]: There's a `require_POST` decorator built in into Django, might be easier to use: https://docs.djangoproject.com/en/1.8/topics/http/decorators/#allowed-http-methods

File: cms/djangoapps/appsembler/views.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        logger.warning('Non-POST request coming to url: /appsemblerstudio')
+        raise Http404
+    try:
+        post_secret = request.POST.get('SecretKey','')
+        print "secret key: {}".format(post_secret)
```
Comment (bezidejni) [inline cms/djangoapps/appsembler/views.py]: Just a note to remove before merging.

**Pull request closed without merging**